pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/kits/KitDisplayNameTest.java

```diff
@@ -15,9 +15,9 @@
 import org.junit.Test;
 
 public class KitDisplayNameTest extends DbTxnBaseTest {
-    private static TestKitUtil
-            testKitUtil = new TestKitUtil("kit_test_instance", "kit_test_instance_guid", "some_prefix", "kit_test_group",
-            KitRequestShippingTest.BLOOD_RNA_KIT_TYPE_NAME);
+    private static TestKitUtil testKitUtil = new TestKitUtil("kit_test_instance", "kit_test_instance_guid",
+            "some_prefix", "kit_test_group", KitRequestShippingTest.BLOOD_RNA_KIT_TYPE_NAME,
+            KitRequestShippingTest.BLOOD_RNA_KIT_TYPE_DISPLAY_NAME);
```

Comment:
@pegahtah I changed this to include the display name because the test was currently passing in `null` for the value and the test was failing.  Can you confirm that this change is appropriate?  I can't tell how the test was passing before since the display name was null and `BLOOD_RNA_KIT_TYPE_DISPLAY_NAME` is different from `BLOOD_RNA_KIT_TYPE_NAME`.